dynamic_array.py

```diff
@@ -0,0 +1,41 @@
+# Hacker Rank Problem Link: https://www.hackerrank.com/challenges/dynamic-array/problem
+
+#!/bin/python3
+
+import os
+import sys
+
+#
+# Complete the dynamicArray function below.
+#
+def dynamicArray(n, queries):

```

Comment:
This is not a super nice design. Hackerrank pushes towards using a single function but this is a lot better handled with a class (__init__ can handle the initialization + some other methods to handle the other parts).